scripts/sync-workflow.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const lr = require('load-runner')();
+const request = require('request');
+
+const requestBodyUtils = require('../util/sync_request_bodies');
+
+const argv = require('yargs')
+      .reset()
```

Comment:
I'm pretty inconsistent with method chains myself! I don't have a strong opinion for this module specifically, but I think it's something to look into for the rest of the codebase for RainCatcher and RHMAP.